bayboone/tests/test_module.py

```diff
@@ -0,0 +1,37 @@
+from unittest import TestCase
+
+#Dependencies of our module
+import numpy as np
+import pandas as pd
+import pymc3 as pm
+
+from bayboone.inference import module
+
+class TestOscillation_model(TestCase):    
+    
+    def test_produces_a_model(self):        
+        test_model = module.oscillation_model(6000, 1)      
+        self.assertTrue(isinstance(test_model, pm.model.Model))    
+        
+    def test_accepts_resonable_inputs(self):        
+        num_numu, num_nue = (10, 22)        
+        #test_model = module.oscillation_model(num_numu, num_nue)       
+        self.assertRaises(ValueError, module.oscillation_model, num_numu, num_nue)    
+        
+class TestInference(TestCase):
```

Comment:
data_io no longer has generative_model, since it was never a generative model. It only simulates data, which I test in test_io.py, so you can get rid of this here.